CefSharp.Core/Internals/JavascriptCallbackImpl.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright � 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#include "Stdafx.h"
+#include "JavascriptCallbackImpl.h"
+#include "Messaging/Messages.h"
+#include "Serialization/Primitives.h"
+#include "Serialization/V8Serialization.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        using namespace Messaging;
+        using namespace Serialization;
+
+        JavascriptCallbackImpl::JavascriptCallbackImpl(JavascriptCallback^ callback, PendingTaskRepository<JavascriptResponse^>^ pendingTasks, WeakReference^ browserWrapper)
```

Comment:
There are a number of places where this isn't the case. I'll clean this up later.